### PR TITLE
release stable linux and mac builds to bintray and depot via travis on release tag push

### PR DESCRIPTION
**Pull request by mwrock**
This PR should completely releases bintray artifacts to stable and hart builds to the public depot when a release tag is pushed.

Also note:
* Mac build is moved to before the sudo unset which should fix the break in master right now.
* This publishes the studio both on unstable and stable releases. I overlooked this detail until seeing it in the release docs just now. I *think* it will work but will know for sure on the master commit before an actual release.
* Updates release docs
* Removes dead windows packaging and vm scripts

Signed-off-by: Matt Wrock <matt@mattwrock.com>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

**Comment by mwrock** [comment]
Fixes #1885

File: .travis.yml [hunk 1]
```diff
@@ -213,11 +215,11 @@ matrix:
           - /hab/studios/home--travis--build--habitat-sh
           - /root/.cargo
       before_install:
-        - ./support/ci/fast_pass.sh || exit 0
-        - if [[ ! -x ./support/ci/deploy_unstable.sh ]]; then chmod +x ./support/ci/deploy_unstable.sh; fi
+        # - ./support/ci/fast_pass.sh || exit 0
```
Comment (smurawski) [inline .travis.yml]: Is this supposed to stay commented?  If so, maybe pull it?

Comment (mwrock) [inline .travis.yml]: no. I'm testing a couple things in the branch. I better tag this "do not merge"

**Comment by mwrock** [comment]
oh cool. say goodbye to that!

**Comment by mwrock** [comment]
I'll push this through after yanking that

**Comment by mwrock** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/209878845).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/209878845).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**